services/api/service.go

```diff
@@ -411,23 +411,28 @@ func (api *RelayAPI) StartServer() (err error) {
 	if err != nil {
 		return err
 	}
-	var foundCapellaEpoch, foundDenebEpoch bool
+
+	api.denebEpoch = -1
+	api.capellaEpoch = -1
 	for _, fork := range forkSchedule.Data {
 		log.Infof("forkSchedule: version=%s / epoch=%d", fork.CurrentVersion, fork.Epoch)
 		switch fork.CurrentVersion {
 		case api.opts.EthNetDetails.CapellaForkVersionHex:
-			foundCapellaEpoch = true
-			api.capellaEpoch = fork.Epoch
+			api.capellaEpoch = int64(fork.Epoch)
 		case api.opts.EthNetDetails.DenebForkVersionHex:
-			foundDenebEpoch = true
-			api.denebEpoch = fork.Epoch
+			api.denebEpoch = int64(fork.Epoch)
 		}
 	}
 
+	if api.denebEpoch == -1 {
+		// log warning that deneb epoch was not found in CL fork schedule, suggest CL upgrade
+		log.Warn("deneb epoch not found in fork schedule, you may need to upgrade the beacon client")
```

Comment:
currently mainnet CL clients don't return a deneb fork schedule, thus i think we should not tell users to upgrade if that's in fact not possible.

```suggestion
		log.Info("deneb epoch not found in fork schedule")
```